ui/packages/app/src/Index.svelte

```diff
@@ -208,6 +208,8 @@
 
 	$: config && (eager || $intersecting[_id]) && load_demo();
 
+	$: console.log(config);
```

Comment:
rm

Reply:
Done and pushed a fix for the case when the path ends in `/`